<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatheterRegistration</class>
 <widget class="QWidget" name="CatheterRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>145</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Segmented catheters</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_SegmentedCatheters">
     <property name="toolTip">
      <string>Pick the input to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLSegmentationNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reconstructed catheters:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLSubjectHierarchyComboBox" name="SubjectHierarchyComboBox_ReconstructedCatheters">
     <property name="defaultText">
      <string>Select folder containing reconstructed catheter models</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="registerButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Distance histogram table:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_DistanceHistogramTable">
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLTableNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Registration error:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_RegistrationError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyComboBox</class>
   <extends>ctkComboBox</extends>
   <header>qMRMLSubjectHierarchyComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
